pkg/controller/master/setting/register.go

```diff
@@ -16,16 +19,26 @@ func Register(ctx context.Context, management *config.Management, options config
 	deployments := management.AppsFactory.Apps().V1().Deployment()
 	controller := &Handler{
 		namespace:       options.Namespace,
+		apply:           management.Apply,
 		settings:        settings,
 		secrets:         secrets,
 		secretCache:     secrets.Cache(),
 		deployments:     deployments,
 		deploymentCache: deployments.Cache(),
+		httpClient: http.Client{
+			Timeout: 30 * time.Second,
+			Transport: &http.Transport{
+				TLSClientConfig: &tls.Config{
+					InsecureSkipVerify: true,
+				},
+			},
+		},
```

Comment:
should we control request over proxy and CA cert if they are specified on the setting?

Reply:
We don't need to specify explicitly and they will be applied.